ggml/src/ggml-opencl/ggml-opencl.cpp

```diff
@@ -4106,10 +4109,18 @@ static void ggml_cl_set_rows(ggml_backend_t backend, const ggml_tensor * src0, c
 
     switch (dst->type) {
         case GGML_TYPE_F32:
-            kernel = backend_ctx->kernel_set_rows_f32;
+            if (src1->type == GGML_TYPE_I64) {
+                kernel = backend_ctx->kernel_set_rows_f32_i64;
+            } else if (src1->type == GGML_TYPE_I32) {
```

Comment:
Compiler complains `ggml/src/ggml-opencl/ggml-opencl.cpp:4334:24: warning: variable 'kernel' is used uninitialized whenever 'if' condition is false [-Wsometimes-uninitialized]`.

Maybe just use `else` here since there is already an assert (line 4069) - types other than `i64` and `i32` will fail the assert.

Everything looks good otherwise.